js/draw/draw.js

```diff
@@ -86,8 +79,14 @@ function getPosition(elem) {
     var yPosition = 0;
       
     while (elem) {
-        xPosition += elem.offsetLeft - elem.scrollLeft + elem.clientLeft;
-        yPosition += elem.offsetTop - elem.scrollTop + elem.clientTop;
+        // || 0 -> for mozilla firefox compatability !!
```

Comment:
I believe elem.offsetLeft and elem.offsetTop are undefined in firefox so when undefined is used in the calculation, the sum is NaN and all the nodes get an x and y of NaN.